rapids-cmake/cython-core/create_modules.cmake

```diff
@@ -117,9 +118,10 @@ function(rapids_cython_create_modules)
     add_custom_command(OUTPUT ${cpp_filename}
```

Comment:
Shouldn't `${depfile}` be here too?

Reply:
Depfile support has been added through https://github.com/scikit-build/cython-cmake/commit/490f1998147af498793f88365b2a9293311a74ee